lnd.go

```diff
@@ -715,9 +719,9 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 					"is synced", res.err)
 			}
 
-			ltndLog.Debugf("Syncing to block timestamp: %v, is "+
-				"synced=%v", time.Unix(res.bestBlockTime, 0),
-				res.synced)
+			ltndLog.DebugS(ctx, "Syncing to block chain",
+				"best_block_time", time.Unix(res.bestBlockTime, 0),
+				"is_synced", res.synced)
```

Comment:
note for reviewers: this is proof that the new custom linter is doing its thing since this line is > 80 chars